ui/react-components/components/AppointmentListSidePanelWrapper/AppointmentListSidePanelWrapper.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from "react";
+import AppointmentListSidePanelSearch from "../AppointmentListSidePanelSearch/AppointmentListSidePanelSearch";
+import AppSpecialityFilter from "../AppSpecialityFilter/AppSpecialityFilter";
+import Label from "../Label/Label";
+import ToggleButton from "../ToggleButton/ToggleButton";
+import { showSelectedContainer } from "./AppointmentListSidePanelWrapper.module.scss";
+import TransformTreeData from "../../utils/AppointmentListSidePanelWrapper/transformTreeData";
+import GetFilteredNodesOnToggle from "../../utils/AppointmentListSidePanelWrapper/getFilteredNodesOnToggle";
+import GetFilteredNodesOnSearch from "../../utils/AppointmentListSidePanelWrapper/getFilteredNodesOnSearch";
+import classNames from "classnames";
+import { useEffect } from "react";
+import { injectIntl } from "react-intl";
+import PropTypes from "prop-types";
+import { getAllServices } from "../../api/__mocks__/serviceApi";
```

Comment:
Is this intentional? Use of `__mocks__` in source code.

Reply:
I have removed this __mocks__ from AppointmentListSidePanelWrapper.jsx .